### ISSUE TITLE
v20 PHP-tidy not enabled by default

### ISSUE THREAD

**Issue by JonBendtsen** [Dolibarr/dolibarr-docker#16]
MAIN_RESTRICTHTML_ONLY_VALID_HTML_TIDY = Undefined   (Recommended: 1)   -   Module "php-tidy" must be enabled (currently:  Not available)

V20 dolibarr image.

**Comment by mathieupotier** [comment]
@JonBendtsen which module are you refering too ?
I don't have this on my instance ...

@creekorful what about having it loaded by default or having an extra ENV_VAR at runtime for enabling it on container run...
I think this image should ship with the minimal loaded by default (to optimise the runtime) and let people to enable php_module (if available) by setting an env_var...
I think about this operation in docker-run init script ... https://github.com/Dolibarr/dolibarr-docker/blob/4facf6427c02571f94771859fa52126f9cf259bd/docker-run.sh#L45

**Comment by JonBendtsen** [comment]
> @JonBendtsen which module are you refering too ? I don't have this on my instance ...

yeah, sorry about not being clear. I think that this is once again:

Home, Admin Tools, About Security page, About PHP or About performance pages

**Comment by JonBendtsen** [comment]
this is from /admin/system/security.php?mainmenu=home&leftmenu=admintools

Home, Admin Tools, About Security

**Comment by creekorful** [comment]
> @creekorful what about having it loaded by default or having an extra ENV_VAR at runtime for enabling it on container run...
> 
> I think this image should ship with the minimal loaded by default (to optimise the runtime) and let people to enable php_module (if available) by setting an env_var...
> 

I think core features would indeed benefit from being directly integrated into the image and should work out of the box.

For other features or specific needs I think this could be a good idea to add such loading mechanism  to keep the image size and attack surface low, but I am quite worried about the implementation "cost". 

What does the two of you think about this feature ? Should it be core or nah?

Cheers
